deploy/materialized_view_identifier.sql

```diff
@@ -6,51 +6,37 @@ begin;
 create materialized view ggircs_swrs.identifier as (
   with import_xml as (
     select _ghgr_import.xml_file as source_xml,
-           _ghgr_import.id         as ghgr_id,
-           _ghgr_import.imported_at
+           _ghgr_import.id         as ghgr_import_id,
     from ggircs_swrs.ghgr_import as _ghgr_import
     order by _ghgr_import.id asc
   )
-  select row_number() over (order by ghgr_id asc) as id,
-         ghgr_id,
-         report_details.swrs_report_id,
-         report_details.swrs_facility_id,
-         identifier_type,
-         identifier_value,
-         -- imported_at,
+  select row_number() over (order by ghgr_import_id asc) as id,
+         ghgr_import_id,
+         swrs_identifier.*,
          row_number() over (
            partition by swrs_facility_id, identifier_type
            order by
-             ghgr_id desc,
+             ghgr_import_id desc,
              identifier_value desc
            ) as swrs_identifier_history_id
   from import_xml,
        xmltable(
-           '/ReportData'
+           '//Identifier'
```

Comment:
@dleard the key context here is the `Identifier` class so that's the context to bind to find type and value

Reply:
So the context. ie: Identenfier, Naics, Organisation etc is where to start with the xmltable